manager/secret_manager.go

```diff
@@ -41,7 +41,19 @@ func (r *ResourceManager) AddSecretKeyValue(namespace string, name string, key s
 	return r.kubeClient.AddSecretKeyValue(namespace, name, key, value)
 }
 
-func (r *ResourceManager) UpdateSecretKeyValue(namespace string, name string, key string, value string) (updated bool, err error) {
+func (r *ResourceManager) UpdateSecretKeyValue(namespace string, name string, data map[string]string) (updated bool, err error) {
```

Comment:
I am thinking this should just take `*model.Secret` as a param instead of separate arguments `name string, data map[string]string`

Reply:
Good point, refactoring.